app.go

```diff
@@ -218,9 +219,21 @@ func (app *appImpl) Execute() int {
 		return 125
 	}
 	if err = app.exec(); err != nil {
+		var status int
+		var execError action.ContainerExecError
+
 		fmt.Fprintln(os.Stderr, "Error:", err)
```

Comment:
The message is actually not an error message, but info about exit status. The error message was output to stdErr by Run environment already.

Reply:
Please, move it to `case 1:`, because we need that error if there was an issue outside running action.